docs/conf.py

```diff
@@ -1,41 +1,74 @@
 #!/usr/bin/env python3
 # -*- coding: utf-8 -*-
+# conf
+# Yellowbrick documentation build config file, created by sphinx-quickstart
```

Comment:
I reorganized the configuration to make it a bit easier to manage. Note though that our configuration file is behind the current version so at some point we may want to do a more thorough review.